extension/src/test/suite/cli/background.ts

```diff
@@ -0,0 +1,7 @@
+import { Logger } from '../../../common/logger'
+import { delay } from '../../../util/time'
+
+delay(1000).then(() => {
```

Comment:
[F] This is a script that will stay alive for a lot longer than it needs to. It is run as a child process in a couple of places.